shell/platform/linux/fl_mouse_cursor_plugin.cc

```diff
@@ -102,6 +116,9 @@ static void fl_mouse_cursor_plugin_dispose(GObject* object) {
     self->view = nullptr;
   }
 
+  if (self->system_cursor_table)
+    g_hash_table_destroy(self->system_cursor_table);
```

Comment:
You can replace this with the simpler `g_clear_pointer(&self->system_cursor_table, g_hash_table_unref)`. This will ensure that the table is set to `NULL` after the destruction. (technically dispose can be called multiple times) and the object can still be accessed.